### DISCUSSION TITLE
14.1.0 revalidateTag/Path doesn't work for me

### DISCUSSION THREAD

**Discussion by CreativeCreature7** [vercel/next.js#61166] [Help]
### Summary

```
  const res = await fetch(HttpRoutes.ACTIVITY, {
    headers: headers(),
    next: {
      tags: [HttpTags.ACTIVITY],
    },
  });
  const data = (await res.json()).data;
```

 I am fetching data inside a server component.
 
 inside a route handler I try to revalidate:

```
export const POST = auth(async (req) => {
  const { title } = await req.json();

  const id = new ObjectId().toHexString();

  if (req.auth) {
    await createActivity({
      id,
      title,
      userId: req.auth.user.id,
    });

    revalidateTag(HttpTags.ACTIVITY);
    console.log("here"); // is printing 

    return NextResponse.json({ message: "Created Activity" }, { status: 200 });
  }

  return NextResponse.json({ message: "Not authenticated" }, { status: 401 });
});
```

it doesn't fetch again.
am I missing something?

**Comment by ciruz** [comment]
For me it's not enough code or context to fully understand the issue. Where is `HttpTags.ACTIVITY` set? This tag might be empty or invalid in the revalidation route. Try to return it as `NextResponse.json` message just to make sure its correct.

How are you triggering this revalidation route? Is it through a CMS webhook, manually via a tool like Postman, or through some other method? When you call the revalidation route, do you receive the HTTP status code `200` or `401` for `Not Authenticated` or maybe another status code?
